whois-api/src/test/java/net/ripe/db/whois/api/sso/AuthServiceClientTestIntegration.java

```diff
@@ -61,4 +85,54 @@ public void get_historical_event_date_time() {
         assertThat(historicalUserDetails.response.results.get(0).eventDateTime, is(LocalDateTime.of(2015, 5, 8, 12, 32, 0)));
     }
 
+
+    // Errors
+    @Test
+    public void get_validate_token_response_null_input_then_error() {
+        assertThat(cacheManager.getCache("ssoValidateToken").get(UUID), is(nullValue()));
+
+        final AuthServiceClientException authServiceClientException = assertThrows(AuthServiceClientException.class, () -> {
+            authServiceClient.validateToken(null);
+        });
+
+        assertThat(authServiceClientException.getMessage(), is("No UUID."));
+        assertThat(cacheManager.getCache("ssoValidateToken").get(UUID), is(nullValue()));
+    }
+
+    @Test
+    public void get_sso_uuid_response_null_input_then_error() {
+        assertThat(cacheManager.getCache("ssoUuid").get(USER_EMAIL), is(nullValue()));
+
+        final AuthServiceClientException authServiceClientException = assertThrows(AuthServiceClientException.class, () -> {
+            authServiceClient.getUuid(null);
+        });
+
+        assertThat(authServiceClientException.getMessage(), is("No UUID."));
```

Comment:
Should be no Username? as you are getting UUID from username so username can be null ? Are the tests passing?

Reply:
you are right, thank you for point this out